pkg/test/framework/components/echo/common/deployment/namespace.go

```diff
@@ -278,3 +160,122 @@ spec:
 
 	return cfg.Apply(apply.NoCleanup)
 }
+
+func CreateDefaultEchoConfigs(cfg Config, t resource.Context) []echo.Config {
```

Comment:
Let's just make this a member function of `Config`. Also, you can probably just drop the `Create` in the name. So the call would be `cfg.DefaultEchoConfigs(t)`.